EIPS/eip-3664.md

```diff
@@ -0,0 +1,417 @@
+---
+eip: 3664
+title: Next generation metaverse NFT
+description: This is a standard for next generation metaverse NFTs, which extends [ERC-1155](./eip-1155.md)
+author: TylerZ (@tylerztl), TY (@nimocat)
+discussions-to: https://ethereum-magicians.org/t/eip-3664-next-generation-game-nft-standard/6738
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-17
+requires: 20, 1155
+---
+
+## Abstract
+In this proposal, we created the next gen component-NFT standard, which makes up for the problem that [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md) can only support unchangeable attributes of NFTs. component-NFT can make the attributes in a NFT have five characteristics: transferable, upgradeable, changeable, combinable and evolvable. By using this standard NFTs in any fields, users can create any valuable dapps based on component-NFT standard.
+
+## Motivation
+In the early days of NFTs, ERC-721 was the main NFT protocol. It focused on differentiating NFTs from FTs. But since the emergence of the Layer 2 and Metaverse, outdated NFT protocols would no longer suffice for NFT standards in the new era. We hope to objectize NFTs, and allow for their upgrades, assembly and disaggregation on the blockchain. We will also let them change with the passage of time so that these fundamental functionalities will endow NFTs with a world of possibilities, creating unparalleled value.
+
+The NFT standard proposed by EIP-3664 enable unlimited disaggregation and assembly. Players can forge modularized NFTs with this proposal, and aggregate modularized NFTs into a new NFT, and let new NFTs combine with each other. On the other hand, NFTs can be disaggregated into component parts and reassembled. These powerful functionalities can sustain many different use cases in the future, such as team configuration in competitive gaming, amendable terms and clauses in financial contracts and honor awarding mechanisms in blockchain games. With the present proposal, you can realize any arbitrary functionality with your NFT. At the same time, the present proposal is an extension protocol of ERC-1155 and ERC-721, implying compatibility with ERC-721 and ERC-1155, and enabling the aggregation of NFTs on old protocols into newly modularized ones. 
+
+## Specification
+
+ERC-3664 extends ERC-1155, divides NFT attributes into five categories:
+
+1. General changeable attributes: usually refers to basic attributes such as ATK and DEF. Its functions include increase or decrease in attribute value.
```

Comment:
What is ATK and DEF?

Reply:
ATK and DEF are a Attack and Defense attributes which are often used in games. EIP-3664 are really suitable for implementing game props based on layer 2 solutions, also quite useful for DeFi tools. I will remove some game related terms in order to make it more general to readers. Thanks